unified_range/models.py

```diff
@@ -1,13 +1,10 @@
-from attr import dataclass
-from typing import List, Tuple
+from typing import List, Tuple, NamedTuple
 
 
-@dataclass
-class Bound:
+class Bound(NamedTuple):
```

Comment:
Out of curiosity: why did you replace the `dataclass` by subclassing a `NamedTuple`?

Reply:
Currently we rely on [attrs](https://pypi.org/project/attrs/) to get the dataclass support, while NamedTuple are just as good for our use case and part of the standard library from python 3.5 - https://docs.python.org/3.5/library/collections.html#collections.namedtuple
https://docs.python.org/3.5/library/typing.html#typing.NamedTuple